### DISCUSSION TITLE
TouchPad Setting memorising

### DISCUSSION THREAD

**Discussion by DeadlyJokerK** [seerge/g-helper#2021] [Ideas]
Hi, I am currently using Flow X16 2022 model.
I am totally Satisfied with the functions and the lightness of G-Helper.
But I have a little problem 
I often turn off the touchpad and using the external mouse while using the laptop, then whenver i restart laptop or when it wakes up from the sleep mode.
The touch pad is on again and I must turn if off again.
could you please add the function which can remember my setting and automatically applying whenever G-Helper is started?

**Comment by seerge** [comment]
@DeadlyJokerK hello, I assume you are talking about "tablet mode" being toggled on and off on X16

Just quit app, add to the beginning  of ``%APPDATA%\GHelper\config.json`` following line 

```
"disable_tablet" : 1,
```

And start app agian

https://github.com/seerge/g-helper/wiki/Power-user-settings#disable-tablet-mode-on-x13x16

**Reply by DeadlyJokerK** [reply]
Hi, Thank you for the answer.
But I am not talking about Talbet mode. I occasionally use the Tablet mode, I am talking about the touchpad mouse on the laptop.
every time It sleeps and on or when I close the lid and open again, I must use fn+f10 to turn off the touchpad to off status again

**Reply by seerge** [reply]
@DeadlyJokerK tablet mode is about toggling touchpad on and off based on the usage scenario of the device :) so app sets touchpad ON automatically.

Add above mentioned line to config, to disable this automation :) and it will stay disabled

**Reply by DeadlyJokerK** [reply]
Thank you so much!
It looks like the problem is solved!
